pkg/controller/resource/deploymentconfig.go

```diff
@@ -0,0 +1,70 @@
+package resource
+
+import (
+	"errors"
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+
+	chaosTypes "github.com/litmuschaos/chaos-operator/pkg/controller/types"
+)
+
+var (
+	gvr = schema.GroupVersionResource{
```

Comment:
Can you name this a little Better? `deploymentConfigGVR`

Reply:
It was there, @chandankumar4 told to change it to gvr

Reply:
gvr is also good, As this variable is already inside the deploymentconfig file

Reply:
`gvr` is too generic, still would be better to change it something suitable.

Reply:
gvr is `GroupVersionResource` still the variable doesn't explain for which resource GVR we have.

Reply:
Then we can name it as `deploymentConfigGVR` earlier it was `dcGVR`

Reply:
Yes, thanks @chandankumar4